can-bus/json/FSM/FSM_rx.json

```
@@ -1,6 +1,8 @@
 {
     "messages": [
         "BMS_Vitals",
-        "Debug_CanModes"
+        "Debug_CanModes",
+        "DCM_LeftInverterCommand",
+        "DCM_RightInverterCommand"
     ]
 }
```

nit: missing newline
```suggestion
}

```